css/src/components/segmented-control.scss

```diff
@@ -0,0 +1,133 @@
+$sc-padding-sm: 0.5em 1em !default;
+$sc-padding: 0.75em 1.25em !default;
+$sc-padding-lg: 1em 1.5em !default;
+
+$sc-font-color: $primary !default;
+$sc-border: $border-width solid $border !default;
+$sc-hover-background-color: $primary-background !default;
+$sc-hover-border: $border-width solid $border !default;
+$sc-hover-border-color: $primary-hover !default;
+$sc-active-hover-background-color: $primary-hover !default;
+$sc-active-hover-border-color: $primary-hover !default;
+$sc-active-background-color: $primary !default;
+$sc-active-border-color: $primary !default;
+$sc-active-color: $text-invert !default;
+
+.segmented-controls {
+	display: grid;
+	justify-content: center;
+	width: 100%;
+	grid-gap: $spacer-5;
+	grid-template-columns: max-content auto max-content;
+	font-size: $font-size-7;
+	white-space: nowrap;
+
+	&.segmented-lg {
+		font-size: $font-size-5;
+	}
+}
+
+.segmented-control-previous,
+.segmented-control-next {
+	display: flex;
+	align-items: center;
+	padding: $sc-padding-sm;
+	border: $sc-border;
+	border-radius: $border-radius-rounded;
+	background-color: $sc-active-background-color;
+	color: $sc-active-color;
+	font-weight: $weight-semibold;
+	cursor: pointer;
+	box-shadow: $box-shadow-medium;
```

Comment:
Open nit question feel free to resolve. Should there be box-shadow on this element?
![image](https://github.com/microsoft/atlas-design/assets/38262647/2d54045c-f0db-4003-94c0-7f416df051f9)

I don't see it in the figma or our other navigation patterns, if not could we remove, if so could we request design to update it to match?
Figma:
![image](https://github.com/microsoft/atlas-design/assets/38262647/b3a5b424-074e-4338-b8fb-64b1b39fd2da)

Reply:
Good call.  This should be removed from the css and then added when using with atomics where necessary.  Thanks @jwmunn

Reply:
Removed the box shadows from the next/prev and the controls

Reply:
Thanks!